Documentation/network/concepts/masquerading.rst

```diff
@@ -181,3 +192,14 @@ by default, use interfaces listed in the ``devices`` field as the egress masquer
 when ``egress-masquerade-interfaces`` is empty. When ``egress-masquerade-interfaces`` is set, 
 it takes precedence over ``devices`` to choose which network interface should perform masquerading.
 You can set ``egress-masquerade-interfaces`` to match multiple interfaces like this: ``eth+ ens+``.
+
+Masquerade traffic to Remote Nodes
```

Comment:
This section now duplicates the one from above - it looks like a leftover, could you remove it?

Reply:
resolved in the new commit